lib/compute/on-chain-status-checker.ts

```
@@ -95,9 +95,10 @@ export class OnChainStatusChecker {
       promises.push(
         (async function (statusChecker: OnChainStatusChecker): Promise<void> {
           try {
-            statusChecker.updateOrder(order)
```

missed the await here so errors weren't being caught and logged as expected. Also catching and swallowing the error causes the promise not to reject which impacted metrics reporting